Src/AutoFixture.xUnit.net2.UnitTest/Scenario.cs

```diff
@@ -72,6 +72,42 @@ public void InlineAutoDataSuppliesDataSpecimensOnlyForNonProvidedValues(string s
             Assert.NotEqual("bar", s3);
         }
 
+        // This test and its associated types is used to document one of the
+        // InlineAutoDataAttribute constructor overloads.
+        [Theory]
+        [MyCustomInlineAutoData(1337)]
+        [MyCustomInlineAutoData(1337, 7)]
```

Comment:
It took me a while to understand why `y` was missing from the test assertions. I wonder if a named constant that indicates we don't care about the value is easier to read?